### PR TITLE
Tra genepop 4.7

### PR DESCRIPTION
**Pull request by tiagoantao**
This upgrades genepop to 4.7
----------------

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.

### CODE REVIEW

**Comment by stain** [comment]
Do we need both 4.6.x and 4.7.x? 

If so then better have two folders for that, I think it's confusing to have one `4.6.9/meta.yml` and one `meta.yml`

**Comment by tiagoantao** [comment]
@stain Sorry for the delay in answering.
I have changed the directory structure.
Version 4.6 is still widely used and 4.6/4.7 and not totally compatible. E.g. Not having 4.6 would break existing versions of Biopython's PopGen module

**Comment by bgruening** [comment]
> If so then better have two folders for that, I think it's confusing to have one 4.6.9/meta.yml and one meta.yml

But this is in general easier for automation. We essentially now without any heuristic where the latest version is stored. Important for any bio.tools integration or any bots.

File: recipes/genepop/4.7.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package:
+  name: genepop
+  version: "4.7.0"
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/genepop/4.7.0/meta.yaml]: This needs to start with 0.

File: recipes/genepop/4.7.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package:
+  name: genepop
+  version: "4.7.0"
```
Comment (bgruening) [inline recipes/genepop/4.7.0/meta.yaml]: The version seems to be wrong, according to the link. Can you please use {{ verssion }} instead.

Comment (bgruening) [reply recipes/genepop/4.7.0/meta.yaml]: Or the link is wrong :)

Comment (tiagoantao) [reply recipes/genepop/4.7.0/meta.yaml]: @bgruening : First my apologies for taking so long.
In this case its neither: after talking with the author, we decided that for existing genepop versions we would take the out from the exinsting R packages. So, the R package is 1.0.5 but Genepop is 4.7.0. People do not really know the R package version.
In the future (hopefully) a standalone version of genepop will be provided (actually it is provided and always was, but the author did not kept old versions, just replaced the source code zip).
All this kerfufle is just to make sure we are able to maintain exact versions (reproducibility and all).
What probably makes sense is add this comment to the yaml, what do you think?

Comment (bgruening) [reply recipes/genepop/4.7.0/meta.yaml]: Oh yes, please add this, otherwise it is very confusing :)

Comment (tiagoantao) [reply recipes/genepop/4.7.0/meta.yaml]: Added the comment to the yaml file.

**Pull request merged**